protocol/0076-DANO-data-node.md

```diff
@@ -186,4 +186,4 @@ It must be possible to add to the data node APIs that return the result of calcu
 1. All events that are emitted on the full unfiltered event stream are processed by the DataNode (no data is lost) (<a name="0076-DANO-018" href="#0076-DANO-018">0076-DANO-018</a>)  
 1. If a DataNode loses connection to a Vega node if will attempt to reconnect and if the cached data received from the Vega node is enough to continue working it can resume being a DataNode. (<a name="0076-DANO-019" href="#0076-DANO-019">0076-DANO-019</a>)  
 1. If the DataNode loses connection to a Vega node and it is unable to reconnect in time to see all the missing data, it will shutdown. (<a name="0076-DANO-020" href="#0076-DANO-020">0076-DANO-020</a>)  
```

Comment:
Should we have a config value for time to reconnect and shutdown after that time has elapsed?

Reply:
Thanks @ettec , in that case , can we please remove AC `0076-DANO-020`

Reply:
done